ci/prometheus-down.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Very minor, but mind running this through `shellcheck`?

Reply:
Sigh.  It hates my old-school style.  I cleaned things up a bit, though.